packages/barclay-integration/src/barclay-integration.ts

```diff
@@ -0,0 +1,3 @@
+export function barclayIntegration(): string {
```

Comment:
Do we need these files? We can just remove this I assume 🤔

Reply:
Removed it in this and the `sezzle-integration` e2e testing PR.